src/models/products.ts

```diff
@@ -49,6 +49,13 @@ export const ProductsModel = sequelize.define("products", {
     type: DataTypes.STRING(128),
     allowNull: true,
   },
+  brand_id :{   // add it 
```

Comment:
don't add unnecessary comments

Reply:
Just to ensure they notice it, then it can be removed.